docs/articles/new-expensify/billing-and-subscriptions/Billing-page-coming-soon.md

```diff
@@ -1,6 +1,49 @@
 ---
 title: Billing and Subscriptions
-description: Coming soon
+description: Billing Page Overview
 ---
 
-# Coming Soon
+# Billing Overview
+
+At the beginning of each month, the Billing Owner for the workspace will be billed for the previous month’s activity. 
+Your Expensify bill is determined by: 
+The number of active members in your workspace
+- Whether you have a Collect or Control plan
+- Whether you’re on pay-per-use or an annual subscription
+- Whether you’re using the Expensify Visa® Commercial Card
```

Comment:
We're not calling it the Expensify Card on the help pages? Actually, it looks like in one of the headings below we use "The Expensify Card"

Reply:
Yes, because this is the first time it's mentioned on this page it needs to be Expensify Visa® Commercial Card - https://stackoverflowteams.com/c/expensify/questions/17349/17350#17350